core/src/costs.rs

```diff
@@ -451,51 +450,64 @@ impl SyscallCosts {
             BlockTimestamp => self.gr_block_timestamp.cost_for_one(),
             Random => self.gr_random.cost_for_one(),
             ReplyDeposit => self.gr_reply_deposit.cost_for_one(),
-            Send(len) => cost_with_per_byte!(gr_send, len),
-            SendWGas(len) => cost_with_per_byte!(gr_send_wgas, len),
+            Send(len) => self.gr_send.cost_for_with_bytes(self.gr_send_per_byte, len),
```

Comment:
I think using macro here is more safe, because we have less ability to make a mistakes in costs pairs naming. You can still use `cost_for_with_bytes` inside macro.

Reply:
Seems good idea

Reply:
Fixed